olive/systems/system_config.py

```diff
@@ -147,7 +147,15 @@ def validate_config_type(cls, values):
         if system_alias_class:
             values["type"] = system_alias_class.system_type
             if system_alias_class.accelerators:
-                values["config"]["accelerators"] = [{"device": acc} for acc in system_alias_class.accelerators]
+                valid_accelerators = []
+                for device in system_alias_class.accelerators:
+                    valid_accelerators.extend(
+                        {"device": acc["device"], "execution_providers": acc.get("execution_providers")}
+                        for acc in values["config"]["accelerators"]
```

Comment:
![image](https://github.com/microsoft/Olive/assets/13343117/1e1fa71d-fb2e-4294-af49-f9153e8267a6)
`values["config"]["accelerators"]` might be None, right? But None is not iterable

Reply:
for aml, the accelerators should not be None and must be provided.